### DISCUSSION TITLE
Editing emqx_auth_mnesia.conf in docker?

### DISCUSSION THREAD

**Discussion by justinbyerss** [emqx/emqx#5571] [General]
Hi,

I am currently running EMQX inside a docker container and I would like my clients to authenticate in order for them to connect.  After reading the documentation it appears I have to edit the emqx_auth_mnesia.conf file, however I am unsure how to edit this file as the file resides inside the docker container.

Does anyone have any experience with this? 
Am I right in assuming that I need to edit the emqx_auth_mnesia.conf file in order to add authentication?

Any help would be appreciated, Thanks.

**Comment by qzhuyan** [comment]
You can either mount the file to the docker container or you can open a shell in the the docker container then edit the file.
